### ISSUE TITLE
Error when performing a development build for iOS

### ISSUE THREAD

**Issue by CodeWithKyrian** [mybigday/whisper.rn#274]
When I want to perform a development build for iOS on a fresh project, I keep getting this error: 

```
› Compiling whisper.rn Pods/whisper-rn » RNWhisper.mm

❌  (node_modules/whisper.rn/ios/RNWhisper.mm:10:9)

   8 | 
   9 | #ifdef RCT_NEW_ARCH_ENABLED
> 10 | #import <RNWhisperSpec/RNWhisperSpec.h>
     |         ^ 'RNWhisperSpec/RNWhisperSpec.h' file not found
  11 | #endif
  12 | 
  13 | @implementation RNWhisper

```

I followed all required steps for installation yet the issue still persists. I ran `xcodebuild clean` in the `ios` folder and tried the build again and it failed. I deleted the folder entirely and tried again and it keeps failing. I also uninstalled the library itself and started the whole steps afresh but this keeps happening. Any idea what could be the issue and how to resolve it please?

**Comment by jhen0409** [comment]
What version of React Native in your project?

**Comment by CodeWithKyrian** [comment]
v0.76.2

**Comment by NoahBakayou** [comment]
Same issue...

**Comment by sooryranga** [comment]
I was able to fix it with the following change - https://github.com/mybigday/whisper.rn/pull/275

**Comment by sooryranga** [comment]
@jhen0409 is it possible do a release with this fix? Thanks!

**Comment by jhen0409** [comment]
> @jhen0409 is it possible do a release with this fix? Thanks!
> 

It already released in 0.4.0-rc.10.

**Comment by sooryranga** [comment]
Thank you!
